libraries/c_sdk/standard/ble/src/iot_ble_mqtt_serialize.c

```diff
@@ -40,14 +40,36 @@
 
 #include "iot_ble_config.h"
 
+#include "mqtt_lightweight.h"
+
 
 /* MQTT internal includes. */
 #include "platform/iot_threads.h"
 #include "iot_serializer.h"
 #include "platform/iot_network_ble.h"
```

Comment:
you could remove this header file dependency, its part of old network interface.